### ISSUE TITLE
Multiple sprites broken

### ISSUE THREAD

**Issue by ZebraFlesh** [cascornelissen/svg-spritemap-webpack-plugin#83]
I just upgraded from 3.3.5 to 3.3.6 and multiple instances of the plugin have stopped working. If you specify two instances, you get no output; comment one of them out and it starts working. Downgrading to 3.3.5 resolves the issue.

**Comment by cascornelissen** [comment]
Looks like I've made an assumption in the `3.3.6` release that doesn't always hold true 🤔  Could you try the latest `master` release and report if that solves your issue?

```shell
npm install --save-dev cascornelissen/svg-spritemap-webpack-plugin#master
```

**Comment by cascornelissen** [comment]
@ZebraFlesh, any updates on this?

**Comment by cascornelissen** [comment]
Just tested this myself and it turns out this issue was related to #82 but had to fixed in a different way which I've done in e3dfddd. It would be great if someone could confirm this fix works, I'll release a new patch version later today anyway.

**Comment by ZebraFlesh** [comment]
Sorry for the delay: this fix works and I'm now getting multiple sprites. Thank you!

**Comment by cascornelissen** [comment]
@ZebraFlesh, no problem at all, thanks for confirming the fix works! It has been released in version [`3.3.7`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.3.7) 🚀
